### PR TITLE
add udatetime recipe

### PR DESCRIPTION
**Pull request by daviskirk**
This recipe adds udatetime a fast RFC3339 compliant python date-time library

### CODE REVIEW

**Comment by daviskirk** [comment]
The upstream package only supports posix systems so I skipped windows in the build recipe.

File: recipes/udatetime/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (igortg) [inline recipes/udatetime/meta.yaml]: Would be nice to remove all the comments from the example recipe (see note above), so the file would be more readable.

**Comment by daviskirk** [comment]
Sorry, I hadn't realized the build had failed.  Updated version and hash to corrected values.

**Comment by synapticarbors** [comment]
@daviskirk -- Thanks for your contribution to conda-forge (and apologies for letting this PR languish). A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Pull request merged**